go/pkg/tool/tool.go

```diff
@@ -393,6 +398,55 @@ func (c *Client) DownloadAction(ctx context.Context, actionDigest, outputPath st
 	return c.writeProto(is, filepath.Join(outputPath, "input_node_properties.textproto"))
 }
 
+func (c *Client) writeExecScript(ctx context.Context, cmd *repb.Command, filename string) error {
+	if cmd == nil {
+		return fmt.Errorf("invalid comment (nil)")
+	}
+
+	var runActionScript bytes.Buffer
+	runActionFilename := filepath.Join(filepath.Dir(filename), "run_command.sh")
+	wd := cmd.WorkingDirectory
+	execCmd := strings.Join(cmd.GetArguments(), " ")
+	runActionScript.WriteString(fmt.Sprintf("#!/bin/bash\n\n"))
+	runActionScript.WriteString(fmt.Sprintf("# This script is meant to be called by %v.\n", filename))
```

Comment:
"by" implies an actor. "from within" is probably clearer?

Reply:
Done.